xesar-connect/src/main/kotlin/com/open200/xesar/connect/extension/XesarConnectComponentExt.kt

```diff
@@ -0,0 +1,100 @@
+package com.open200.xesar.connect.extension
+
+import com.open200.xesar.connect.Topics
+import com.open200.xesar.connect.XesarConnect
+import com.open200.xesar.connect.messages.command.FindComponent
+import com.open200.xesar.connect.messages.command.Query
+import com.open200.xesar.connect.messages.command.RemoteDisengage
+import com.open200.xesar.connect.messages.command.RemoteDisengagePermanent
+import com.open200.xesar.connect.messages.event.FindComponentPerformed
+import com.open200.xesar.connect.messages.event.RemoteDisengagePerformed
+import com.open200.xesar.connect.messages.event.RemoteDisengagePermanentPerformed
+import com.open200.xesar.connect.messages.query.EvvaComponent
+import com.open200.xesar.connect.messages.query.QueryList
+import java.util.*
+import kotlinx.coroutines.Deferred
+
+/**
+ * Enable/disable the beeping signal to find the component.
+ *
+ * @param installationPointId The ID of the installation point
+ * @param enable Enable or disable the beeping signal
+ * @param requestConfig The request configuration (optional).
+ */
+suspend fun XesarConnect.findComponent(
```

Comment:
should we move this to InstalltionPoint since it refers to a installationPointId.